RELEASE.md

```diff
@@ -83,3 +86,56 @@ The usual steps are:
     * Copy the bundle files from `github.com/kuadrant/kuadrant-operator/tree/v0.W.Z/bundle`
     * Copy `github.com/kuadrant/kuadrant-operator/tree/v0.W.Z/bundle.Dockerfile` with the proper fix to the COPY commands
       (i.e. remove /bundle from the paths)
+
+--- 
+
+## Release file format.
+
+This example of the `release.toml` file uses tag `v1.0.1` as reference.
+
+```toml
+# FILE: ./release.toml
+[kuadrant]
+channels = ["alpha",]
+release = "1.0.1"
+
+[dependencies]
+Authorino_bundle = "0.16.0"
+Console_plugin = "0.0.14"
+DNS_bundle = "0.12.0"
+Limitador_bundle = "0.12.1"
+Wasm_shim = "0.8.1"
+
+```
+
+```sh
+git checkout v1.0.1
+cat make/release.mk
+
+# FILE: ./make/release.mk
+#Release default values
+AUTHORINO_OPERATOR_VERSION=0.16.0
+AUTHORINO_OPERATOR_BUNDLE_IMG=quay.io/kuadrant/authorino-operator-bundle:v0.16.0
+LIMITADOR_OPERATOR_VERSION=0.12.1
+LIMITADOR_OPERATOR_BUNDLE_IMG=quay.io/kuadrant/limitador-operator-bundle:v0.12.1
+DNS_OPERATOR_VERSION=0.12.0
+DNS_OPERATOR_BUNDLE_IMG=quay.io/kuadrant/dns-operator-bundle:v0.12.0
+WASM_SHIM_VERSION=0.8.1
+RELATED_IMAGE_WASMSHIM=oci://quay.io/kuadrant/wasm-shim:v0.8.1
+RELATED_IMAGE_CONSOLEPLUGIN=quay.io/kuadrant/console-plugin:v0.0.14
+IMG=quay.io/kuadrant/kuadrant-operator:v1.0.1
+BUNDLE_VERSION=1.0.1
+BUNDLE_IMG=quay.io/kuadrant/kuadrant-operator-bundle:v1.0.1
+CATALOG_IMG=quay.io/kuadrant/kuadrant-operator-catalog:v1.0.1
+CHANNELS=alpha
+BUNDLE_CHANNELS=--channels=alpha
+VERSION=1.0.1
```

Comment:
I guess it's not expected for us to touch these? Does this serve a purpose here ?

Reply:
Yeah, this is here as a reference for now. I expect by the time we are done that file will not exist.